requirements.in

```diff
@@ -59,7 +59,7 @@ posthoganalytics==3.5.0
 psycopg2-binary==2.9.7
 PyMySQL==1.1.1
 psycopg[binary]==3.1.18
-pyarrow==15.0.0
+pyarrow==17.0.0
```

Comment:
Any breaking changes with 2 new major versions of pyarrow?

Reply:
Not on dw end. DLT with delta requires it now

@tomasfarias note here incase of batch exports impact

Reply:
Removed these updates altogether. Realized they weren't needed. Also, note that the DLT version upgrade causes a test to fail. Errors about s3 configuring missing